strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/configuration/ServerConfigurationControllerTestIT.java

```diff
@@ -232,6 +232,45 @@ void serverSettingsShouldBeSaved(String acceptHeader)
 
     }
 
+
+    @ParameterizedTest
+    @ValueSource(strings = { MediaType.APPLICATION_JSON_VALUE,
+                             MediaType.TEXT_PLAIN_VALUE })
+    @WithMockUser(authorities = { "CONFIGURATION_SET_BASE_URL",
+                                  "CONFIGURATION_SET_PORT" })
+    void serverSettingsShouldAllowEmptyURL(String acceptHeader)
+    {
+        String baseUrl = "";
+
+        Integer port = 18080;
+
+        // assign settings to server
+        ServerSettingsForm serverSettingsForm = new ServerSettingsForm(baseUrl, port, "Strongbox-1234");
+
+        String url = getContextBaseUrl() + "/serverSettings";
+
+        // Save the form
+        mockMvc.log().all().contentType(MediaType.APPLICATION_JSON_VALUE)
+               .accept(acceptHeader)
+               .body(serverSettingsForm)
+               .when()
+               .post(url)
+               .peek() // Use peek() to print the output
+               .then()
+               .statusCode(HttpStatus.OK.value()) // check http status code
+               .body(containsString(SUCCESSFUL_SAVE_SERVER_SETTINGS));
```

Comment:
Cosmetic: If the methods are chained, keep them one per line for legibility:
```suggestion
        mockMvc.log()
               .all()
               .contentType(MediaType.APPLICATION_JSON_VALUE)
               .accept(acceptHeader)
               .body(serverSettingsForm)
               .when()
               .post(url)
               .peek() // Use peek() to print the output
               .then()
               .statusCode(HttpStatus.OK.value()) // check http status code
               .body(containsString(SUCCESSFUL_SAVE_SERVER_SETTINGS));
```